public/app/plugins/panel/canvas/editor/options.ts

```diff
@@ -205,4 +207,25 @@ export const optionBuilder: OptionSuppliers = {
       defaultValue: { value: LineStyle.Solid, label: 'Solid' },
     });
   },
+
+  addDataLinks: (builder, context) => {
+    const category = ['Data links'];
+    builder
+      .addCustomEditor({
+        category,
+        id: 'enableOneClick',
+        path: 'oneClickLinks',
+        name: 'One-click',
+        description: 'When enabled, the top action or link in the list below works with a single click',
```

Comment:
don't have actions yet for this release so may want to update wording here

```suggestion
        description: 'When enabled, the top link in the list below works with a single click',
```